src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -948,6 +952,7 @@ export class BrightScriptDebugSession extends BaseDebugSession {
             v.type = result.type;
             v.evaluateName = result.evaluateName;
             v.frameId = frameId;
+            v.presentationHint = result.presentationHint ? { kind: result.presentationHint } : undefined;
```

Comment:
Adds the presentationHint to every variable if specified. This is what helps fade out the `[[children]]` variables in vscode. We created a shorthand since it's a bit verbose.